pkg/validation.go

```diff
@@ -147,6 +156,11 @@ func (*Validation) externalLink(link Link) (Link, error) {
 
 			resp.Body.Close()
 			break
+		} else if resp.StatusCode == http.StatusTooManyRequests {
+			status = false
+			message = "Too many requests"
+			v.limiter.Limit()
```

Comment:
Maybe we could try using the `Retry-After` header if provided, and fallback to default backoff otherwise?

Reply:
It's out of the scope of this issue. You're welcome to create ticket for this ;)

Reply:
Well, I wouldn't say it's out of the scope of the issue but if it's too much work and we aren't sure how much value it will bring I agree that it's not necessary to implement it now.